data/json/items/armor/jewelry.json

```diff
@@ -1123,6 +1151,17 @@
     "use_action": [ "MEDITATE" ],
     "flags": [ "NO_WEAR_EFFECT" ]
   },
+  {
+    "id": "wooden_hair_beads",
+    "copy-from": "garnet_silver_tiara",
+    "type": "ARMOR",
+    "name": { "str": "wooden hair beads", "str_pl": "sets of wooden hair beads" },
+    "description": "An assortment of wooden beads used to decorate locs or braids.",
+    "material": [ "wood" ],
```

Comment:
Do you need to adjust weight and volume? I imagine a metal tiara weighs and takes more volume than hair beads.

Reply:
Might need to, That was an oversight on my part, apologies!